pipelines/calibrate.py

```diff
@@ -1,32 +1,15 @@
 # Config override for lsst.pipe.tasks.calibrate.CalibrateTask
 # This config is for the ap_verify datasets that use gaia/panstarrs as their
 # refcat names.
-from lsst.meas.algorithms import LoadIndexedReferenceObjectsTask
 
 # Use gaia for astrometry (phot_g_mean for everything, as that is the broadest
-# band with the most depth)
-# Use panstarrs for photometry (grizy filters)
-for refObjLoader in (config.astromRefObjLoader,
-                     config.photoRefObjLoader,):
-    refObjLoader.retarget(LoadIndexedReferenceObjectsTask)
+# band with the most depth).
+
 config.connections.astromRefCat = "gaia"
 config.astromRefObjLoader.ref_dataset_name = config.connections.astromRefCat
-config.astromRefObjLoader.filterMap = {
-    "u": "phot_g_mean",
-    "g": "phot_g_mean",
-    "r": "phot_g_mean",
-    "i": "phot_g_mean",
-    "z": "phot_g_mean",
-    "y": "phot_g_mean",
-    "VR": "phot_g_mean"}
+config.astromRefObjLoader.anyFilterMapsToThis = "phot_g_mean"
+config.astromRefObjLoader.filterMap = {}
```

Comment:
These two lines are technically not necessary, since [DM-27013](https://jira.lsstcorp.org/browse/DM-27013) is done.

Reply:
I didn't know that, but for configuration that depends on the input data, I prefer to be explicit instead of trusting that the defaults match up with I really need.